library/ssl_tls.c

```diff
@@ -7081,6 +7095,11 @@ static int ssl_write_real( mbedtls_ssl_context *ssl,
     }
     else
     {
+        /*
+         * The user is trying to send a message the first time, so we need to
+         * copy the data into the internal buffers and setup the data structure
+         * to keep track of partial writes
```

Comment:
Thanks for this additional documentation :+1:

Nitpicking: There are full stops missing at the end of this and the previous documentation snippet.

Reply:
I think I will leave it as it is. I think the reason for this is that the doxygen documentation should be well punctuated, etc because it will ultimately end up as a formal documentation website. However, these are only in code comments, if you look at the other comments like this, you will see that (1) the style is fairly inconsistent across the library and (2) a lot of them don't have a full stop at the end either.

Reply:
Please correct it in case you add an additional commit for the other documentation fix.